src/core/annotation.js

```diff
@@ -602,6 +602,14 @@ var WidgetAnnotation = (function WidgetAnnotationClosure() {
       this.setFlags(AnnotationFlag.HIDDEN);
     }
 
+    // Support ResetForm Action Widget
+    if (data.fieldType === 'Btn' && dict.get('A') !== undefined) {
```

Comment:
Try `isDict(dict.get('A'))` instead of `dict.get('A')`?

Maybe even with storing `dict.get('A')` in a separate variable if you're going to use the value often.